template-helpers/dashboard.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const { LocaleUtils } = require("./../locale-utils");
+const { makeBreachCards } = require("./breaches");
+
+const getBreachesForEachEmail = (args) => {
+  const verifiedEmails = args.data.root.verifiedEmails;
+  const locales = args.data.root.req.supportedLocales;
+
+  verifiedEmails.forEach(email => {
+    const breachCards = makeBreachCards(email.breaches, locales);
+    email.breaches = breachCards;
+  });
+  return verifiedEmails;
+};
+
+
+const getUserPreferences = (args) => {
+  const unverifiedEmails = args.data.root.unverifiedEmails;
+  const verifiedEmails = args.data.root.verifiedEmails;
+  const primaryEmail = verifiedEmails.shift();
+
+  const locales = args.data.root.req.supportedLocales;
+  args.data.root.preferences = true;
+
+  const emailAddresses = {
+    primary: {
+      subhead: "Firefox Account Email - Primary",
```

Comment:
Assuming this will come in a future commit, but these strings need to be localizable.